src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,27 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+
+class Centro(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (!vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        } else throw Exception("El vendedor ya esta incluido")
+    }
+
+//mal, revisar
+    fun vendedorEstrella() =
+        vendedores.map { v -> v.puntajeCertificaciones() }.max()
```

Comment:
Para casos así podés usar el método `maxBy`

```suggestion
    fun vendedorEstrella() = vendedores.maxBy { it.puntajeCertificaciones() }
```